server/src/main/scala/com/redi/scalademo/application/Main.scala

```diff
@@ -0,0 +1,43 @@
+package com.redi.scalademo.application
+
+import org.eclipse.jetty.server.Server
+import org.eclipse.jetty.servlet.{DefaultServlet, ServletContextHandler, ServletHolder}
+import org.eclipse.jetty.util.resource.Resource
+import org.glassfish.jersey.server.ServerProperties
+import org.glassfish.jersey.servlet.ServletContainer
+
+object Main extends App {
+
+  val context = new ServletContextHandler(ServletContextHandler.SESSIONS)
+  context.setContextPath("/")
+
+  val jettyServer = new Server(8080)
+  jettyServer.setHandler(context)
+
+  val jerseyServlet: ServletHolder =
+    context.addServlet(classOf[ServletContainer], "/api/*")
+  jerseyServlet.setInitOrder(0)
+
+  val defaultServlet: ServletHolder =
+    context.addServlet(classOf[DefaultServlet], "/")
+  val publicPath: String =
+    Resource.newResource(ClassLoader.getSystemResource("public"))
+      .getURI
+      .toASCIIString
```

Comment:
OK, I just googled about this method's value. I still feel weird because URLs should support UTF-8 characters, but it seems for our purpose ```toASCIIString``` should be okay.